inst/@sym/vertcat.m

```diff
@@ -124,7 +123,7 @@
 %! q = sym(ones(0, 2));
 %! assert (isequal ([v; q], v))
 
-%!error <ShapeError>
+%!error <ShapeError: `self` and `bott` must have the same number of columns.>
```

Comment:
Ok.

Just FYI for next time: its not necessary to quote the exact error: if SymPy makes a minor change then I don't need the test to fail.  The idea is it should fail with ShapeError rather than e.g., a "syntax error".